docs/source/install/rhel6.rst

```diff
@@ -62,9 +75,12 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
     sudo chkconfig mongod on
     sudo chkconfig rabbitmq-server on
 
-    # Install and configure postgres 9.4
-    sudo yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-1.noarch.rpm
-    sudo yum -y install postgresql94-server postgresql94-contrib postgresql94-devel
+    # Install and configure postgres 9.4. Based on OS type install the ``redhat`` one or ``centos`` one.
+    # RHEL:
+    if grep -q "Red Hat" /etc/redhat-release; then sudo yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-2.noarch.rpm; fi
+
+    # CentOS:
+    if grep -q "CentOS" /etc/redhat-release; then sudo yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-redhat94-9.4-2.noarch.rpm; fi
 
```

Comment:
PS. If you've read original instructions of mine. The postgress thing was mentioned there...